client/components/LearnerViewDirective.js

```diff
@@ -584,12 +581,7 @@ tie.directive('learnerView', [function() {
             var syntaxErrorIndex = feedback.getSyntaxErrorIndex();
```

Comment:
Okay, per our conversation, I removed the index altogether and just am doing a simple forloop to check if there's a syntax error. Also renamed the function to be more specific.